unit_conversion/unit_conversion.py

```diff
@@ -365,18 +370,20 @@ def __init__(self, unitType):
 
     def __str__(self):
         return "The unit type: %s is not in the UnitConversion database"%self.unitType
-    
+
 class MismatchedUnitError(UnitConversionError):
     """
     Exception raised when a unit is not in the Unitconversion database
 
     """
     ##fixme -- this appear to be used anywhere...
-    def __init__(self, (FromUnit, FromUnitType, ToUnit, ToUnitType) ):
-        self.FromUnit     =  FromUnit    
+    def __init__(self, FromUnit_FromUnitType_ToUnit_ToUnitType):
```

Comment:
same as previous comment -- I like the old way, 'cause it makes the API clearer...